src/components/MutationTumorTypeFrequencyDecomposition.tsx

```diff
@@ -31,7 +31,6 @@ class MutationTumorTypeFrequencyDecomposition extends React.Component<ITumorType
         ): (
             <MutationTumorTypeFrequencyTable
                 data={this.data}
-                hugoSymbol={this.props.hugoSymbol}
```

Comment:
remove `hugoSymbol` because it is not using in `MutationTumorTypeFrequencyTable`
https://github.com/knowledgesystems/signal/blob/master/src/components/MutationTumorTypeFrequencyTable.tsx

And `MutationTumorTypeFrequencyTable` now is in the `react-variant-view`